app/models/partner.rb

```diff
@@ -211,6 +211,26 @@ def export_json
     }
   end
 
+  def families_served
+    families.count
+  end
+
+  def children_served
+    children.count
+  end
+
+  def family_zipcodes
```

Comment:
For these methods I'd add `_count` at the end to ensure there isn't any confusion about if this is supposed to output a number of the collection of children that the partner serves.